bitcode/devicelib.cl

```diff
@@ -39,36 +39,46 @@
 #pragma OPENCL EXTENSION cl_khr_int64_base_atomics : enable
 #pragma OPENCL EXTENSION cl_khr_int64_extended_atomics : enable
 
-EXPORT unsigned /* long */ long int __chip_umul64hi(unsigned /* long */ long int x, unsigned /* long */ long int y) {
-    unsigned /* long */ long int mul = (unsigned /* long */ long int)x * (unsigned /* long */ long int)y;
-    return (unsigned /* long */ long int)(mul >> 64);
+#ifndef __opencl_c_generic_address_space
+#error __opencl_c_generic_address_space needed!
+#endif
+
+EXPORT unsigned /* long */ long int
+__chip_umul64hi(unsigned /* long */ long int x,
+                unsigned /* long */ long int y) {
+  unsigned /* long */ long int mul =
+      (unsigned /* long */ long int)x * (unsigned /* long */ long int)y;
+  return (unsigned /* long */ long int)(mul >> 64);
```

Comment:
It would be better if the changes unrelated to this patch were not included here. It's hard spot issues through the mass style-formatting.

Reply:
Yep, I often do a separate clang-format commit, but in this case it was so small diff, I incldued it here.